### PR TITLE
Create creating-wordpress-stack-with-orchestration.md

### PR DESCRIPTION
**Pull request by BadCrumpany**

### CODE REVIEW

**Comment by rackernexus** [comment]
Build 1189 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-3a1b93b3d/how-to/). valid for 14 days

**Comment by rackernexus** [comment]
Build 1190 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-39077b7dc/how-to/). valid for 14 days

**Comment by rackernexus** [comment]
Build 1192 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-39077b7dc/how-to/). valid for 14 days

**Comment by csilva235** [comment]
I just wanted to make a note about this article as @BadCrumpany submitted it on my behalf since we hadn't setup my GitHub access when it was completed. 

This article is a complete step by step guide to setup and scale out a Wordpress stack with Orchestration. There is an article that vaguely talks about this, but a customer with minimal knowledge in Linux may not know what all happens with the Orchestration stack for Wordpress. This is the current Wordpress Orchestration article:

https://support.rackspace.com/how-to/deploy-wordpress-packages-by-using-rackspace-cloud-orchestration/

The problem is that the article doesn't explain that it's using NGINX to deploy Wordpress or really explain much about what's going on. Additionally, the current article doesn't mention scaling out or link to any information about scaling out.  I think the idea of scaling out easily should be readily apparently as part of the stack process. The article that does talk about scaling is lacking in a few ways. This is the article:

https://support.rackspace.com/how-to/how-to-scale-multinode-wordpress-by-using-rackspace-orchestration/

With the scaling article, the problem is that the lsync setup described does not work with a default stack deployment.  If the customer builds out a stack and then wants to scale, the lsync commands in the article assume the customer created a user called wp_user and deployed an SSH key and set all of that stuff up before trying to scale. None of those tasks are mentioned in the initial build or the scaling article. It's also pretty confusing with the lsync path since it's specifically for lsyncing a single domain rather than the /var/www/ directory to grab all virtual hosts they may create. 


I created the proposed document because our current documentation assumes the customer knows all the ins and outs of the orchestration stack, how Wordpress works, and how to scale out at a later time. It doesn't talk about why get get a LB or Cloud DB and how to use them. 

@BadCrumpany  and I also discussed the idea of possibly splitting off the scaling section of this article to it's own article and linking in as a reference for customers who want to scale after the build. 

TL;DR - I think the article submitted can work as a beginner friendly, start to finish, Wordpress stack setup. It goes over all the parts of the stack and explains the concept of scaling and how to do it. I think if the current articles remain, they will need a bit of tweaking to maybe include a link to the full setup for customer who may be new to building this environment as well as fix the incorrect lsync info or specify the criteria needed to use that guide. 

Feel free to reach out if you have any questions.

**Comment by csilva235** [comment]
I'm not sure how to modify the file that was submitted, but I noticed that the version of PHP it shows to be installing is PHP 7.4, but it should actually be PHP 7.3.

**Comment by rackernexus** [comment]
Build 1207 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-305b71cc8/how-to/). valid for 14 days

**Comment by rackernexus** [comment]
Build 1208 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-d5dc0273c/how-to/). valid for 14 days

**Comment by rackernexus** [comment]
Build 1215 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-d4d6e7221/how-to/). valid for 14 days

File: content/cloud-orchestration/use-rackspace-orchestration-to-build-a-wordpress-stack.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+        excludeFrom="/etc/lsyncd/lsyncd.exclude.lua",
+        rsync = {
+                compress = true,
+                acls = true,
+                verbose = true,
+                rsh = "/usr/bin/ssh -p 22 -o StrictHostKeyChecking=no",
+                _extra = {"-a"}
+        	}
+	}
+6. Save and exit the file.
```
Comment (chri2547) [inline content/cloud-orchestration/use-rackspace-orchestration-to-build-a-wordpress-stack.md]: Should the remaining text in this article be italicized?

Comment (catlook) [reply content/cloud-orchestration/use-rackspace-orchestration-to-build-a-wordpress-stack.md]: hmmm prolly not.  let me see if i can escape th _ without messing up the code sample.

**Comment by rackernexus** [comment]
Build 1217 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-9d94d064f/how-to/). valid for 14 days
